.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,64 @@
+name: UnitTests
+
+on:
+    - pull_request
+    - push
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php }} + Composer ${{ matrix.composer-flags }} + Symfony@${{ matrix.symfony || '*' }}"
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                include:
+                    -   php: '5.5'
+                        composer-flags: '--prefer-lowest'
+                    -   php: '5.5'
+                        symfony: '^3.0'
+                    -   php: '5.6'
+                    -   php: '7.0'
+                    -   php: '7.1'
+                        symfony: '^4.0'
+                    -   php: '7.2'
+                        symfony: '^5.0'
+                    -   php: '7.3'
+                    -   php: '7.4'
+                    -   php: '8.0'
+                        symfony: '^6.0'
```

Comment:
you may need sth like this https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6095/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR102-R106